services/lfs/server.go

```diff
@@ -179,6 +179,12 @@ func BatchHandler(ctx *context.Context) {
 		return
 	}
 
+	batchSize := len(br.Objects)
```

Comment:
LFSClient.BatchSize hasn't been used.

Reply:
it's used the line below actually `batchSize > setting.LFSServer.MaxBatchSize`, but you're right that it's only used once. I was considering whether to put a debug line to print the batch size the client requested. Would you prefer:
* leave as-is
* add such a debug line
* remove this var and just get the length in-line for that same check

Reply:
remove the lfs.client and the configuration items if no place use that.

Reply:
okay, I think I probably don't want a log line, so I've moved this inline. We can add it back if we want logging later.